Enterprise-CMCS/macpro-mdct-mfp
# Add ensureAvailableReport to Cypress commands

### Description


This adds a cypress command to make sure that any e2e test that fails can reset itself and be able to run again without having to clean anything out of the DB or start a whole new PR. It also enables the dashboard and WP e2e test to run as part of the Git Action

### Related ticket(s)


CMDCT-3502

---
### How to test

Note the password is the same for logging in as a state user or admin user in MFP. If you don't have the password or think its been updated, check the proj-cms-mdct-devs channel for the pinned message at the top called MDCT App Access

- Navigate to tests/cypress
- Run `CYPRESS_ADMIN_USER_PASSWORD='insertpasswordhere!' CYPRESS_STATE_USER_PASSWORD='Insertpasswordhere!' yarn test`
- Click e2e configuration
- Run the new tests! 

---
### Pre-review checklist



---
